frontend/src/pages/org/archived-item-detail/ui/qa.ts

```diff
@@ -1,216 +1,718 @@
-      <btrix-desc-list horizontal>
-        <btrix-desc-list-item label=${msg("Review Result")}>
-          ${when(
-            reviewStatus !== undefined,
-            () =>
-              reviewStatus
-                ? html`<span class="capitalize"
-                    >${crawlReviewStatus(reviewStatus)}</span
-                  >`
-                : msg("No review"),
-            renderEmpty,
-          )}
-        </btrix-desc-list-item>
-        <btrix-desc-list-item label=${msg("Analysis Status")}>
-          ${when(
-            qaRuns,
-            (qaRuns) =>
-              qaRuns[0] ? runAnalysisStatus(qaRuns[0]) : msg("No Analysis"),
-            renderEmpty,
-          )}
-        </btrix-desc-list-item>
-        <btrix-desc-list-item label=${msg("Analysis Elapsed Time")}>
-          ${when(
-            qaCrawlExecSeconds !== undefined,
-            () =>
-              qaRuns?.[0] && qaCrawlExecSeconds
-                ? humanizeExecutionSeconds(qaCrawlExecSeconds)
-                : msg("N/A"),
-            renderEmpty,
+                  if (!downloadLink) {
+                    console.debug("no download link");
+                    return;
+                  }
+
+                  downloadLink.loading = true;
+                  const file = await this.getQARunDownloadLink(run.id);
+                  if (file) {
+                    downloadLink.disabled = false;
+                    downloadLink.href = file.path;
+                  } else {
+                    downloadLink.disabled = true;
+                  }
+                  downloadLink.loading = false;
+                }}
+              >
+                <sl-menu>
+                  ${run.state === "canceled"
+                    ? nothing
+                    : html`
+                        <btrix-menu-item-link href="#" download>
+                          <sl-icon name="download" slot="prefix"></sl-icon>
+                          ${msg("Download QA Run")}
+                        </btrix-menu-item-link>
+                        <sl-divider></sl-divider>
+                      `}
+                  <sl-menu-item
+                    @click=${() => {
+                      this.deleting = run.id;
+                      void this.deleteQADialog?.show();
+                    }}
+                    style="--sl-color-neutral-700: var(--danger)"
+                  >
+                    <sl-icon name="trash3" slot="prefix"></sl-icon>
+                    ${msg("Delete Item")}
+                  </sl-menu-item>
+                </sl-menu>
+              </btrix-overflow-dropdown>
+            </div>
+          </btrix-table-cell>
+        </btrix-table-row>
+      `,
+    );
+  };
+
+  private readonly renderDeleteConfirmDialog = () => {
+    const runToBeDeleted = this.qaRuns?.find((run) => run.id === this.deleting);
+
+    if (!runToBeDeleted) return;
+
+    return html`
+      <btrix-dialog id="deleteQARunDialog" .label=${msg("Delete QA Run?")} open>
```

Comment:
```suggestion
      <btrix-dialog id="deleteQARunDialog" .label=${msg("Delete QA Run?")}>
```

Reply:
Another comment for this one, we've been calling these both "QA Runs", "Analysis Runs", and "Crawl Analysis".  I'm partial to referencing it as "Crawl Analysis" as I think it requires the least additional explanation, what are the units of crawl analysis?  When the "Crawl Analysis" process completes, what does it output?

Whatever that is, we should refer to it here!  Some options:
- Crawl Analysis Runs: "delete crawl analysis run"
- Crawl Analysis: "delete crawl analysis"
- Crawl Analysis Instance: "delete crawl analysis instance"

Reply:
Could be "delete analysis run" since the list is labeled "Analysis Runs"? I just renamed the download button to match.

Reply:
Addressed in 2bc2e0e